lib/wallaby/httpclient.ex

```diff
@@ -23,18 +23,22 @@ defmodule Wallaby.HTTPClient do
   def request(method, url, params, [{:encode_json, false} | _]) do
     make_request(method, url, params)
   end
+  def request(method, url, params, [{:max_try, max} | _]) do
+    make_request(method, url, params, {0, max})
+  end
```

Comment:
this adds a capability to add an arbitrary maximum try limit, right? Imo, we don't need that and it's also not called from anywhere, old behaviour should be fine?

Reply:
What is a imo?

Reply:
An imo is an `in my opinion` relatively common internet jargon, sorry :)